src/olympia/reviewers/tests/test_models.py

```diff
@@ -1315,6 +1315,22 @@ def test_check_is_locked(self):
         self.addon.update(type=amo.ADDON_LPAPP)
         assert AutoApprovalSummary.check_is_locked(self.version) is False
 
+    def test_check_is_pending_rejection(self):
+        assert AutoApprovalSummary.check_is_pending_rejection(self.version) is False
+
+        flags = VersionReviewerFlags.objects.create(version=self.version)
+        assert AutoApprovalSummary.check_is_pending_rejection(self.version) is False
+
+        flags.update(
+            pending_rejection=datetime.now() + timedelta(hours=1),
+            pending_rejection_by=user_factory(),
+            pending_content_rejection=False,
+        )
+        assert AutoApprovalSummary.check_is_pending_rejection(self.version) is True
+
+        flags.update(pending_content_rejection=True)
```

Comment:
I'm checking that changing the `pending_content_rejection` field has no effect - `check_is_pending_rejection()` should continue to return `True` in that case. We could split it up but I don't think it matters since that's the only change - this style is consistent with the other tests in that class

Reply:
Okay so the test is actually that updating content rejection to true should NOT have an effect. Okay. It wasn't super clear in the test but it makes sense now that you mention. Maybe worth a comment for future clarity but up to you.